warehouse/models/mart/payments/_payments.yml

```diff
@@ -119,3 +119,54 @@ models:
       - name: transaction_date_pacific
       - name: day_of_week
         description: Day of week value of the transaction in the range [1,7] with Sunday as the first day of the week.
+  - name: fct_elavon__transactions
+    description: Transactions processed by Elavon
+    columns:
+      - name: payment_reference
+      - name: payment_date
+      - name: account_number
+      - name: routing_number
+      - name: fund_amt
+      - name: batch_reference
+      - name: batch_type
+      - name: customer_batch_reference
+      - name: customer_name
+      - name: merchant_number
+      - name: external_mid
+      - name: store_number
+      - name: chain
+      - name: batch_amt
+      - name: amount
+      - name: surchg_amount
```

Comment:
For these things that are abbreviated, not sure if it's worth specifying at least what the abbreviation stands for (even if it's mostly pretty obvious). Also would be nice to either define these columns a bit more or link to other documentation (if available).